badge_server/badge_server.py

```diff
@@ -334,8 +337,7 @@ def one_badge_image():
 
     status, _, _, _ = _get_all_results_from_cache(package_name)
     color = STATUS_COLOR_MAPPING[status]
-    package_name = _sanitize_package_name(package_name)
```

Comment:
can you explain why _sanitize_package_name() is only called if badge_name is None instead of every time?

Reply:
Good catch! I was thinking if badge_name is None, use the package_name as the badge_name. But it could be possible that the user specified badge_name also needs to be sanitized. Fixed.